adapters/eplanning/eplanning.go

```diff
@@ -212,6 +218,68 @@ func (adapter *EPlanningAdapter) MakeRequests(request *openrtb2.BidRequest, reqI
 	return requests, errors
 }
 
+func unmarshalSupplyChain(req *openrtb2.BidRequest) *openrtb2.SupplyChain {
```

Comment:
It may make things clearer to just pass `unmarshalSupplyChain` the `request.Source.Ext` object rather than the whole `request` object. Makes it clear what `unmarshalSupplyChain` is actually doing and using.